jazkarta/shop/browser/checkout.py

```diff
@@ -24,13 +32,318 @@
 from ..vocabs import country_names
 
 
-@implementer(IStripeEnabledView)
 class CheckoutForm(BrowserView):
+
+    def __call__(self):
+        payment_processor = get_setting('payment_processors')
+        if payment_processor != []:
+            p_p = payment_processor[0]
```

Comment:
Yeah, but I'm wondering why we allow the admin to enable more than one if only one will actually be used.